src/index.js

```diff
@@ -0,0 +1,32 @@
+import './index.css';
+import Utility from './utility.js';
+import Interactive from './interactive.js';
+
+const inputForm = document.getElementById('to-do-form');
+const input = document.getElementById('to-do-input');
+const editInputForm = document.getElementById('editTodo-form');
+const editInput = document.querySelector('.edit-item');
+
```

Comment:
Impressive the way you just imported files from the module. Awesome.

Reply:
Thank you @Moise-code  for reviewing my code!